Cargo.toml

```diff
@@ -8,7 +8,7 @@ exclude = ["assets/*", ".github"]
 
 
 [dependencies]
-nalgebra = "*"
+nalgebra = "0.29.0"
```

Comment:
why this version?

Reply:
supports conversion to euler angle directly (it's the most recent version atm)